pkg/authorize/kubecertmanager/authorize.go

```diff
@@ -70,11 +71,40 @@ func (c *authorizer) Admit(_ context.Context, cx *api.Context) field.ErrorList {
 	return c.validateIngress(cx, ingress)
 }
 
-// validateIngress checks the the image complys with policy
-func (c *authorizer) validateIngress(cx *api.Context, ingress *extensions.Ingress) field.ErrorList {
+// validateIngress checks that the ingress's host resolve to the hostname of the cluster's internal or external ingress
+func (c *authorizer) validateIngressPointed(cx *api.Context, ingress *extensions.Ingress) field.ErrorList {
+	var errs field.ErrorList
+
+	// @step: get namespace for this object
+	namespace, err := utils.GetCachedNamespace(cx.Client, cx.Cache, ingress.Namespace)
+	if err != nil {
+		return append(errs, field.InternalError(field.NewPath("namespace"), err))
+	}
+
+	enableDNSCheck := true
+	if v, found := namespace.GetAnnotations()[cx.Annotation(EnableDNSCheck)]; found {
```

Comment:
```Go
enableDNSCheck := namespace.GetAnnotations()[cx.Annotation(EnableDNSCheck)] == "true"
```

Reply:
much simpler!
I've amended it to
```
	enableDNSCheck := namespace.GetAnnotations()[cx.Annotation(EnableDNSCheck)] != "false"
```
As the current behaviour is to take a rubbish value as true (so it's only disabled if the flag is set to "false")